code/game/gamemodes/events/biomass.dm

```diff
@@ -202,10 +202,11 @@
 		var/area/Hallway = locate(type)
 
 		for(var/turf/simulated/floor/Floor in Hallway.contents)
-			if(Floor.contents.len <= 0)
+			if(Floor.contents.len <= 1)//There's always an /atom/movable/lighting_overlay on turfs

```

Comment:
This solution is highly suspect and breaking prone. Using Dylan's density check would be superior